README.md

```diff
@@ -28,7 +28,15 @@ pip install -r requirements.txt
 # Try it out
 python manim.py example_scenes.py OpeningManimExample
 ```
-
+### Mac OSX
+1. Install [FFmpeg],[LaTeX],[cairo](if you only want to run on CPU) in commandline using homebrew.
```

Comment:
Now cairo is only used in manim to make svg files for `Text`, and will not be used for rendering. OpenGL will be used anyway.
And adding square brackets to these dependencies is not a markdown syntax, it will show these [], which is not what we want.